client/a8c-for-agencies/sections/overview/overview.tsx

```diff
@@ -11,6 +11,7 @@ import MobileSidebarNavigation from 'calypso/a8c-for-agencies/components/sidebar
 import OverviewBody from './body';
 import OverviewHeaderActions from './header-actions';
 import OverviewSidebar from './sidebar';
+import './style.scss';
```

Comment:
[Non-blocker] We usually keep one line space before we import the style file

Reply:
Thank you!